Codeinwp/neve
# Feat/gradient color picker

### Summary
Add Gradient control to colors

### Will affect visual aspect of the product

YES

### Screenshots 

### Test instructions


1. Inside Customizer on controls that are tied to background or overlay the gradient options should be available. See image:
![image](https://user-images.githubusercontent.com/23024731/162488337-513244f7-27dc-4ed8-a6ac-3a44562a46f6.png)
2. Check that controls respect the gradient used, inside Customizer and on the Frontend
3. Here is a list of controls that are affected by this:

- Button's background (Primary and Secondary)
- Layout Single Post, all controls that use a boxed layout (eg. comments), Cover Background
- Archive, all controls that use a boxed layout, Card Background, Covers Overlay
- Search component from HFG background
- HFG row backgrounds
- All components from HFG that allow background color change from the Style tab.
- Form Fields background


Closes #2472.
